.claude/CLAUDE.md

```
@@ -0,0 +1,260 @@
+# 특정 테스트 메서드 실행
+./gradlew test --tests "akuma.whiplash.domains.alarm.domain.service.AlarmCommandServiceTest.RingAlarmTest.success"
+
+# Docker Compose로 전체 스택 실행 (MySQL, Redis, Prometheus, Grafana, k6)
+docker-compose up -d
+```
+
+## 아키텍처
+
+### 레이어 구조 (도메인별로 반복)
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)